src/tests/ftest/util/test_utils_pool.py

```diff
@@ -897,6 +935,61 @@ def drain(self, rank, tgt_idx=None):
 
         return status
 
+    def get_acl(self):
+        """Get ACL from a DAOS pool.
+
+        Raises:
+            CommandFailure: if the control method is API.
+
+        Returns:
+            str: dmg pool get-acl output.
+
+        """
+        if self._dmg_ready("get_acl()"):
+            return self.dmg.pool_get_acl(pool=self.identifier)
+
+        return None
```

Comment:
If the `self._dmg_ready` only raises exceptions when the dmg cannot be run, this method would simplify to:
```suggestion
        self._dmg_ready("get_acl()")
        return self.dmg.pool_get_acl(pool=self.identifier)
```